android/jni/com/mapswithme/maps/editor/Editor.cpp

```diff
@@ -356,4 +356,10 @@ Java_com_mapswithme_maps_editor_Editor_nativeCreateNote(JNIEnv * env, jclass cla
   Editor::Instance().CreateNote(MercatorBounds::FromLatLon(lat, lon), jni::ToNativeString(env, text));
 }
 
+// static boolean nativeIsHouseValid(String houseNumber);
+JNIEXPORT jboolean JNICALL
+Java_com_mapswithme_maps_editor_Editor_nativeIsHouseValid(JNIEnv * env, jclass clazz, jstring houseNumber)
```

Comment:
В C/C++ есть ворнинг на не используемые параметры.

> On Apr 7, 2016, at 13:50, Dmitry Yunitsky <notifications@github.com> wrote:
> 
> In android/jni/com/mapswithme/maps/editor/Editor.cpp <https://github.com/mapsme/omim/pull/2810#discussion_r58852608>:
> 
> > @@ -356,4 +356,10 @@ Java_com_mapswithme_maps_editor_Editor_nativeCreateNote(JNIEnv * env, jclass cla
> >    Editor::Instance().CreateNote(MercatorBounds::FromLatLon(lat, lon), jni::ToNativeString(env, text));
> >  }
> >  
> > +// static boolean nativeIsHouseValid(String houseNumber);
> > +JNIEXPORT jboolean JNICALL
> > +Java_com_mapswithme_maps_editor_Editor_nativeIsHouseValid(JNIEnv * env, jclass clazz, jstring houseNumber)
> Я не согласен, что это явное улучшение. В JNI два первых аргумента всегда идут в комплекте, это привычно и понятно всем, кто пишет такой код.
> … <x-msg://266/#>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> 
>  <https://github.com/mapsme/omim/pull/2810#discussion_r58851155> <https://github.com/mapsme/omim/pull/2810/files/94e4930e51d14e01f30778e20b0549e6286b8fed#r58851155>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2810/files/94e4930e51d14e01f30778e20b0549e6286b8fed#r58852608>